src/FakeItEasy/FakeFacade.cs

```diff
@@ -50,6 +51,7 @@ public virtual void ClearRecordedCalls(object fakedObject)
             manager.ClearRecordedCalls();
         }
 
+        [Obsolete("Test fixture initialization will be removed in version 5.0.0.")]

```

Comment:
This method isn't publicly visible, so I don't think it needs the `Obsolete` attribute.